selfhost/codegen.jakt

```diff
@@ -1147,60 +1147,51 @@ struct CodeGenerator {
     function codegen_enum_debug_description_getter(mut this, enum_: CheckedEnum) throws -> String {
         mut output = ""
 
-        output += "ErrorOr<String> debug_description() const { "
-        output += "auto builder = TRY(StringBuilder::create());"
-        output += "TRY(this->visit("
+        output += "ErrorOr<String> debug_description() const {\n"
+        output += "auto builder = TRY(StringBuilder::create());\n"
+        output += "switch (this->index()) {"
 
-        mut variant_lambdas: [String] = []
-        for variant in enum_.variants.iterator() {
+        for i in (0..enum_.variants.size()).exclusive() {
```

Comment:
This is exclusive by default

Reply:
Sure,
Will fix that up in stage0 as well (by hand....)